pkg/antctl/handlers/version.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/vmware-tanzu/antrea/pkg/monitor"
+)
+
+var _ Factory = new(Version)
+
+// VersionResponse describes the response of the version command.
+type VersionResponse struct {
+	AgentVersion      string `json:"agentVersion,omitempty" yaml:"agentVersion,omitempty"`
+	ControllerVersion string `json:"controllerVersion,omitempty" yaml:"controllerVersion,omitempty"`
+	AntctlVersion     string `json:"antctlVersion,omitempty" yaml:"antctlVersion,omitempty"`
```

Comment:
This should not replied from the server, and so should not be defined in VersionResponse?

Reply:
In the `definition.go`, the `versionTransform` will populate the `antctl` version to it.

Reply:
`omitempty` makes sure the serialization result would not contains keys with empty values.

Reply:
I saw you populate antctl version, but again where we define fields not implemented by the server to the response message? Here we might just include a single version in the response message definition (for controller it is the controller version and for agent it is the agent version)?

Reply:
OK, got your point, I have updated the code and shrunk the VersionResponse.